cdi/embedded/src/main/java/org/infinispan/cdi/InfinispanExtensionEmbedded.java

```diff
@@ -192,6 +194,68 @@ public void registerCacheConfigurations(CacheManagerEventBridge eventBridge, Ins
 
    }
 
+   /**
+    * The default embedded cache configuration can be overridden by creating a producer which
+    * produces the new default configuration. The configuration produced must have the scope
+    * {@linkplain javax.enterprise.context.Dependent Dependent} and the
+    * {@linkplain javax.enterprise.inject.Default Default} qualifier.
+    *
+    * @param beanManager
+    * @return a custom bean
+    */
+   private Bean<Configuration> createDefaultEmbeddedConfigurationBean(BeanManager beanManager) {
+      return new BeanBuilder<Configuration>(beanManager).beanClass(InfinispanExtensionEmbedded.class)
+            .addTypes(Object.class, Configuration.class)
+            .scope(Dependent.class)
+            .qualifiers(DefaultLiteral.INSTANCE, AnyLiteral.INSTANCE)
+            .beanLifecycle(new ContextualLifecycle<Configuration>() {
+               @Override
+               public Configuration create(Bean<Configuration> bean,
+                     CreationalContext<Configuration> creationalContext) {
+                  return new ConfigurationBuilder().build();
```

Comment:
Please ignore this comment, it should be OK since `GlobalConfiguration` is produced in `createDefaultEmbeddedCacheManagerBean`.